Handbook Update: trade show section (#3093)
FlowFuse/website: src/handbook/customer/marketing/tradeshow.md

---
navTitle: Marketing
navTitle: Trade Shows
navGroup: Sales & Marketing
---

# Trade show
# Trade Shows  

## As exhibitor
## As an Exhibitor  

### Booth design
### Six Months Before the Event  

> The design of the booth is imporant. From the angles that visitors walk past the

Reviewer: This information is now lost?

Author: I've marked it as a draft; I'm going to re-work it and re-request your review when it's ready.

>     - At least one team member should be responsible for building the demo. Ensure the team includes a technically skilled person to support this.  

Reviewer: ```suggestion
    - One team member is responsible for building the demo. Ensure the team member is technically skilled to support this.  
```

>     - Create a Slack channel with the confirmed attendees. The name of the channel should have the following format `#proj-[event-name]-year`

Reviewer: ```suggestion
    - Create a Slack channel with the confirmed attendees. The name of the channel should have the following format `#proj-[event-name]-[year] - e.g. #proj-hannover-25`
```

  - **Accommodation** – book a hotel near the venue, ideally with breakfast included.  
    - If the demo involves physical devices, the team member transporting them should arrive **two days** before the event. The rest of the team should arrive **one day** before.  

### Four Months Before the Event  

- **Define what will be demonstrated**, as this impacts the **booth layout and messaging**.  
  - The CTO should always be involved in demo planning, even if they are not attending the event.  
- **Research booth vendors** and obtain at least **three different quotations**.  

>   - Keep in mind that **US vendors do not include taxes in their quotations**. For example, in Dallas, TX, taxes were around **8%**.  

Reviewer: ```suggestion
  - Keep in mind that US vendors do not include taxes in their quotations. For example, in Dallas, TX, taxes were around **8%**.  
```

Reviewer: More generally, why the overuse of bold? Can you ask your LLM to use less?

## Preparing for a Trade Show as an Exhibitor  

Planning a trade show well in advance is crucial. From aligning on the goals and messaging, to making sure we have the right hardware, giveaways, and booth setup—it’s a team effort. We break this down into five areas:

1. **Logistics** – budget, team selection, travel, accommodation, vendors, equipment and giveaways.
2. **Demo** – what we showcase during the event.
3. **Marketing and Sales** – messaging, promotion and lead generation and nurturing.
5. **Design** – booth layout and artworks.
6. **On-site team** – prep for the event and etiquette.

### Logistics

After the decision has been made to attend an event, the first thing that's needed is a budget. This will guide the next steps in the event planning. The CEO is responsible for setting the budget and it should be tracked in a spreadsheet alongside all the logistic details, such as hotel details, team arrival and departure times, plus any relatet matter and added to the relevant folder in the Drive inside [Fairs](https://drive.google.com/drive/folders/1kfbJy6H3DXYlKDd0RZx-psK7ghDmN4BI?usp=drive_link).

> **The planning should start at least six months in advance**, so there's enough time to book hotels and for the team to block their agendas. The on-site team should be proposed by the CEO and CTO, and in general, the team will be of 4 or 5 members and composed of:

Reviewer: Size depends on the event I think. Like the Mayker event we'll likely send one or two.

- One or two sales person.
- A technically skilled member that will be responsible for setting up and dismantling the demo at the booth.
- One from customer success.
- Someone from any other area that will attend to learn or bring support where needed.

Create a Slack channel with the proposed team and confirm their availability. The name of the channel should have the following format `#proj-[event-name]-[year]` - e.g. `#proj-hannover-25`. Use this channel to communicate any updates related to the trade show, it will be especially useful during the event as well.

After the team has confirmed their availability, the accomodation should be booked considering:

> - Hotels near the venue are preferred over centric ones.

Reviewer: ```suggestion
- Hotels near the venue are preferred over city-central ones.
```
